sys/net/gnrc/transport_layer/tcp/gnrc_tcp_option.c

```diff
@@ -61,14 +63,15 @@ int _option_parse(gnrc_tcp_tcb_t *tcb, tcp_hdr_t *hdr)
                 break;
 
             default:
-                DEBUG("gnrc_tcp_option.c : _option_parse() : Unknown option found.\
-                      KIND=%"PRIu8", LENGTH=%"PRIu8"\n", option->kind, option->length);
-                return -1;
+                if (TCP_OPTION_LENGTH_MIN <= opt_left) {
```

Comment:
This confused me, because it is the other way around below. I'd prefer
```suggestion
                if (opt_left >= TCP_OPTION_LENGTH_MIN) {
```